hudi-common/src/main/java/org/apache/hudi/common/config/HoodieCommonConfig.java

```diff
@@ -71,6 +71,14 @@ public class HoodieCommonConfig extends HoodieConfig {
           + " operation will fail schema compatibility check. Set this option to true will make the newly added "
           + " column nullable to successfully complete the write operation.");
 
+  public static final ConfigProperty<String> ADD_NULL_FOR_DELETED_COLUMNS = ConfigProperty
```

Comment:
Why add a config? If schema evolution is enabled then do this by default?

Reply:
What would the behavior be when this is false and schema evolution is enabled? Is there an option where it would auto-drop the column in the target table?

Reply:
+1 for this to be default behavior or just fail the write operation.

Reply:
If we make this default behavior, pipelines that are designed to fail with dropped columns will start ingesting data with new code. Lets make an explicit callout in the docs/release notes if we want to make this default behavior.